Makefile

```diff
@@ -54,6 +54,10 @@ export DOCKER_DEVKIT_AWS_ARGS ?= \
 	--env AWS_DEFAULT_REGION \
 	--volume "$(HOME)/.aws":"/home/$(USER_NAME)/.aws"
 
+export DOCKER_DEVKIT_GCP_ARGS ?= \
+	--env GOOGLE_APPLICATION_CREDENTIALS \
+	--volume "$(HOME)/.gcp":"/home/$(USER_NAME)/.gcp"
```

Comment:
do we need to mount `.gcp` if `GOOGLE_APPLICATION_CREDENTIALS` is exported?

Reply:
`GOOGLE_APPLICATION_CREDENTIALS` is just a pointer to a file that exist in your system. It has to be mounted.